ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```
@@ -1248,6 +1248,35 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-lifecycle-advanced
+- as: aws-rosa-private-link-stage-f7
+  cron: 31 13 1,8,15,24 * *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      CHANNEL_GROUP: nightly
+      E2E_RUN_TAGS: '@rosa'
+      NAME_PREFIX: ocpe2e
+      REGION: us-east-1
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;ROSA&
```

```suggestion
      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~NonHyperShiftHOST&;~MicroShiftOnly&;ROSA&
```
The workflow lauch a rosa hcp cluster, right? If yes please add `~NonHyperShiftHOST&` exclude the cases which are not suitable for hcp.

---

aws-rosa-private-link-stage-f7  is a classic cluster.The following: rosa-hcp-pl is for hcp

---

Got it, thanks, please update all hcp jobs using `~NonHyperShiftHOST&` in `TEST_FILTERS`.